build/RELEASING.md

```diff
@@ -0,0 +1,84 @@
+
+# Release Process
+
+Releasing of projects from the opentelemetry-dotnet-contrib repo is
+currently a mix of automated workflow and some manual steps. Hopefully
+most of this will be automated soon.
+
+## Pre-requisites
+
+To carry out the release successfully, make sure you fulfill the following:
+
+1. You must be a maintainer on the repo. This will allow you to push tags to
+`main` branch and create a GitHub release later on.
+
+2. Have access to the Nuget token for releasing the project to Nuget.org
+OpenTelemetry account.
+
+## Pre-step
+
+Before starting with the release process, ensure that the latest commit on
+the `main` branch is the commit which has added/updated the Changelog to
+the project being released. This is the commit that will be tagged on the
+release.
+
+## Steps
+
+1. Create and push git tag for the project and the version of the project
+you want to release.
+
+    ```powershell
+    git tag -a <PROJECT TAG PREFIX><VERSION> -m "<PROJECT TAG PREFIX><VERSION>"
+    ```
+
+    You can find the project tag prefix in the workflow file for the project.
```

Comment:
the source of truth should be the .csproj. example : https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Contrib.Instrumentation.Elasticsearch/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.csproj#L6